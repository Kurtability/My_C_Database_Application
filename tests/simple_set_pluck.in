pluck a 1
set a 1 2 3
pluck a 4
pluck a 0
pluck a 3
get a
set a 1 2 3 4 5 6
pluck a 1
pluck a 1
pluck a 2
pluck a 2
pluck a 2
get a
pluck a 1
get a
bye
