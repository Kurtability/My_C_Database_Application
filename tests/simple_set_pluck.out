> pluck a 1
no such key

> set a 1 2 3 
ok

> pluck a 4
index out of range

> pluck a 0
index out of range

> pluck a 3
3

> get a 
[1 2]

> set a 1 2 3 4 5 6
ok

> pluck a 1
1

> pluck a 1
2

> pluck a 2
4

> pluck a 2
5

> pluck a 2
6

> get a 
[3]

> pluck a 1
3

> get a 
[]

> bye
bye
